examples/landmarks/js/visua11y.js

```diff
@@ -825,7 +825,7 @@ function nameFromAttributeIdRefs(element, attribute) {
   if (value.length) {
     idRefs = value.split(' ');
 
-    for (i = 0; i < idRefs.length; i++) {
+    for (var i = 0; i < idRefs.length; i++) {
```

Comment:
`var i` is already declared with the other variables in this function block on line `820`. Could you remove that declaration (on 820) and keep the one you added in the for block?

Reply:
@smhigley done!